README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Automated resources for Vagrant box creation
```

Comment:
We may remove "Automated" part from this as it sounds odd, WDYT ?

Reply:
@DilanUA, this is part of [this](https://github.com/wso2-incubator/vagrant-is/pull/4) pull request. I will move this content to [this](https://github.com/wso2-incubator/vagrant-is/pull/4) pull request, once this is merged.

Reply:
I would suggest "Automation of Vagrant box creation". WDYT?